### PR TITLE
PPTP-1657 : Skeleton for Groups Post Reg Amendment

### PR DESCRIPTION
**Pull request by RyanKellyHMRC**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -48,6 +48,8 @@ lazy val scoverageSettings: Seq[Setting[_]] = Seq(
     "test\\..*",
     ".*(BuildInfo|Routes|Options).*",
     "logger.*\\(.*\\)",
+    ".*.controllers.amendment.group\\..*",
```
Comment (liampan) [inline build.sbt:51]: do we have a ticket/todo to remind us to undo this?

**Review by liampan — APPROVED** [review]

File: build.sbt (hunk 1 above)
Comment (RyanKellyHMRC) [reply build.sbt:51]: Good call. I'll add a TODO in there.

Comment (RyanKellyHMRC) [reply build.sbt:51]: Actually, I'm going to split these up into individual exclusions so that they can be reintroduced piecemeal.

Comment (liampan) [reply build.sbt]: nice touch! will make that much easier :)

**Review by paulblakehmrc — APPROVED** [review]
LGTM

**Pull request merged**